csharp/Sdk.Examples/Horizon/Tutorials/ProcessHistory/ProcessHistory.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Finbourne.Horizon.Sdk.Api;
+using Finbourne.Horizon.Sdk.Client;
+using Finbourne.Horizon.Sdk.Model;
+using NUnit.Framework;
+using Sdk.Examples.Horizon.Utilities;
+
+namespace Sdk.Examples.Horizon.Tutorials.ProcessHistory
+{
+    [TestFixture]
+    public class ProcessHistory : TutorialBase
+    {
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+
+        }
+
+        [OneTimeTearDown]
+        public void TearDown()
+        {
+
+        }
+
+        [Test]
+        public void Get_Latest_Runs()
```

Comment:
this needs to run after we've created the events, set the order using  `[Test, Order(2)]` with the one that creates as  `[Test, Order(1)]`